js/node/node-backend.js

```diff
@@ -82,7 +83,7 @@ async function withThreadPool(run) {
 async function initThreadPool() {
   if (!isMainThread) return;
   workersReady = new Promise((resolve) => (workersReadyResolve = resolve));
-  await wasm.initThreadPool(getEfficientNumWorkers(), filename);
+  await wasm.initThreadPool(workers.numWorkers ?? os.cpus().length, filename);
```

Comment:
Instead of `os.cpus().length`, a better default could be `os.availableParallelism() - 1` https://nodejs.org/api/os.html#osavailableparallelism

I thought the `-1` makes sense because these workers are started in addition to the main thread 🤔

Reply:
Ahhhhh! In that case, I will switch it back to `-1`. I didn't understand that the first thread also did work so I figured it would have extra time for a worker. (I only added `-1` because it ended up being faster in practice for me.) But if I add `-1` back in, it should still work perfectly for a machine with only a single core?

Reply:
> But if I add -1 back in, it should still work perfectly for a machine with only a single core?

ah, great question lol. probably not. maybe `Math.max(n - 1, 1)`?